DataLayer/Repositories/IGameBoardRepository.cs

```diff
@@ -1,6 +1,16 @@
-﻿namespace GameBoard.DataLayer.Repositories
+﻿using System.Threading.Tasks;
+using GameBoard.DataLayer.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace GameBoard.DataLayer.Repositories
 {
     public interface IGameBoardRepository
     {
+        DbSet<ApplicationUser> ApplicationUsers { get; set; }
+        DbSet<Friendship> Friendships { get; set; }
```

Comment:
I'm not sure, but I guess we don't need a public setter on these in the interface.

Reply:
Fixed.